docs/azurefunction_maven.md

```diff
@@ -0,0 +1,25 @@
+# Configure Maven for VS Code
```

Comment:
Hmm, maybe the name here still needs refinement. I think it could be better if we change it to `Configure Maven for Java Azure Functions Development`